<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>admin</class>
 <widget class="QWidget" name="admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1400</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1400</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1400</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>物业管理员</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 127);</string>
  </property>
  <widget class="QListWidget" name="listWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>60</y>
     <width>231</width>
     <height>781</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(240, 240, 240);</string>
   </property>
   <property name="lineWidth">
    <number>1</number>
   </property>
   <property name="midLineWidth">
    <number>0</number>
   </property>
   <property name="spacing">
    <number>5</number>
   </property>
   <item>
    <property name="text">
     <string>     菜单</string>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
   </item>
   <item>
    <property name="text">
     <string>   【】人事管理</string>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>170</red>
       <green>170</green>
       <blue>127</blue>
      </color>
     </brush>
    </property>
   </item>
   <item>
    <property name="text">
     <string>        【】人员信息</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>        【】人员添加</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>        【】信息修改</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>        【】请假审核</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>   【】系统管理</string>
    </property>
    <property name="background">
     <brush brushstyle="SolidPattern">
      <color alpha="255">
       <red>170</red>
       <green>170</green>
       <blue>127</blue>
      </color>
     </brush>
    </property>
   </item>
   <item>
    <property name="text">
     <string>        【】初始化</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>        【】系统参数</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string/>
    </property>
   </item>
  </widget>
  <widget class="QFrame" name="frame">
   <property name="geometry">
    <rect>
     <x>-1</x>
     <y>0</y>
     <width>1401</width>
     <height>61</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0);</string>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_4">
    <property name="geometry">
     <rect>
      <x>22</x>
      <y>7</y>
      <width>170</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Adobe Arabic&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>用户名</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_5">
    <property name="geometry">
     <rect>
      <x>1200</x>
      <y>7</y>
      <width>170</width>
      <height>50</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 75 16pt &quot;Adobe Arabic&quot;;
background-color: rgb(0, 0, 0);
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>退出</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tableView">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>140</y>
     <width>1168</width>
     <height>651</height>
    </rect>
   </property>
   <property name="layoutDirection">
    <enum>Qt::LeftToRight</enum>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 127);
border-color: rgb(0, 0, 0);</string>
   </property>
   <property name="editTriggers">
    <set>QAbstractItemView::NoEditTriggers</set>
   </property>
   <property name="tabKeyNavigation">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QFrame" name="frame_2">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>61</y>
     <width>1168</width>
     <height>36</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton_3">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>5</y>
      <width>93</width>
      <height>28</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
border:none;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QFrame" name="frame_3">
   <property name="geometry">
    <rect>
     <x>233</x>
     <y>98</y>
     <width>1168</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="frameShape">
    <enum>QFrame::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Raised</enum>
   </property>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>5</y>
      <width>93</width>
      <height>31</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>1</y>
      <width>113</width>
      <height>35</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background:transparent;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>5</y>
      <width>51</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;</string>
    </property>
    <property name="text">
     <string>姓 名：</string>
    </property>
   </widget>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>260</x>
     <y>180</y>
     <width>301</width>
     <height>271</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 36pt &quot;Adobe Arabic&quot;;
background-color: rgb(0, 0, 127);
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>欢迎使用！</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>680</y>
     <width>450</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 127);
color: rgb(255, 255, 255);</string>
   </property>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>20</y>
      <width>72</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;
background:transparent;</string>
    </property>
    <property name="text">
     <string>序号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>20</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;
background:transparent;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>20</y>
      <width>90</width>
      <height>41</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 14pt &quot;黑体&quot;;
background-color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>准假</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_4" native="true">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>240</y>
     <width>411</width>
     <height>431</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_35">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>140</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;
background:transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>性别：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_36">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>200</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;
background:transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>联系电话：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_37">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>260</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;
background:transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>身份证号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_38">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>320</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;
background:transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>职位：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_33">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>140</y>
      <width>150</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color: rgb(255, 255, 255);
font: 12pt &quot;黑体&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_34">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>200</y>
      <width>150</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color: rgb(255, 255, 255);
font: 12pt &quot;黑体&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_35">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>260</y>
      <width>150</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color: rgb(255, 255, 255);
font: 12pt &quot;黑体&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_39">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>20</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;
background:transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>姓名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_36">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>20</y>
      <width>150</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color: rgb(255, 255, 255);
font: 12pt &quot;黑体&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_40">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;
background:transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>年龄：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_37">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>320</y>
      <width>150</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color: rgb(255, 255, 255);
font: 12pt &quot;黑体&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_23">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>370</y>
      <width>100</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_38">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>150</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color: rgb(255, 255, 255);
font: 12pt &quot;黑体&quot;;</string>
    </property>
   </widget>
  </widget>
  <widget class="QWidget" name="widget_5" native="true">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>240</y>
     <width>411</width>
     <height>431</height>
    </rect>
   </property>
   <widget class="QLabel" name="label_41">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>180</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;
background:transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>性别：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_42">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>230</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;
background:transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>联系电话：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_43">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>280</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;
background:transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>身份证号：</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_44">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>330</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;
background:transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>职位：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_39">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>130</y>
      <width>150</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color: rgb(255, 255, 255);
font: 12pt &quot;黑体&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_40">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>180</y>
      <width>150</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color: rgb(255, 255, 255);
font: 12pt &quot;黑体&quot;;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_41">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>230</y>
      <width>150</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color: rgb(255, 255, 255);
font: 12pt &quot;黑体&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_45">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>80</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;
background:transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>姓名：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_42">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>30</y>
      <width>150</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color: rgb(255, 255, 255);
font: 12pt &quot;黑体&quot;;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_46">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>130</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;
background:transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>年龄：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_43">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>280</y>
      <width>150</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color: rgb(255, 255, 255);
font: 12pt &quot;黑体&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_24">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>370</y>
      <width>100</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>修改</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_44">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>80</y>
      <width>150</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color: rgb(255, 255, 255);
font: 12pt &quot;黑体&quot;;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_25">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>370</y>
      <width>100</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>查询</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_47">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>30</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 12pt &quot;黑体&quot;;
background:transparent;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>工号：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_45">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>330</y>
      <width>150</width>
      <height>32</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background:transparent;
color: rgb(255, 255, 255);
font: 12pt &quot;黑体&quot;;</string>
    </property>
   </widget>
  </widget>
  <widget class="QTableView" name="tableView_2">
   <property name="geometry">
    <rect>
     <x>598</x>
     <y>240</y>
     <width>441</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 14pt &quot;黑体&quot;;
color: rgb(255, 255, 255);
background-color: rgb(0, 0, 127);</string>
   </property>
  </widget>
  <widget class="QWidget" name="widget_6" native="true">
   <property name="geometry">
    <rect>
     <x>600</x>
     <y>240</y>
     <width>411</width>
     <height>431</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 127);</string>
   </property>
   <widget class="QPushButton" name="pushButton_26">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>290</y>
      <width>100</width>
      <height>42</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>黑体</family>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);
background-color: rgb(0, 0, 127);</string>
    </property>
    <property name="text">
     <string>初始化</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>110</y>
      <width>311</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 25pt &quot;Adobe Arabic&quot;;
background:transparent;
color: rgb(0, 0, 0);</string>
    </property>
    <property name="text">
     <string>系统数据初始化</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
